### PR TITLE
Fix deploy without kogito.yml file

### PR DESCRIPTION
**Pull request by ljmotta**
- Remove the requirement of a kogito.yml file to deploy the service.
- Fix the Windows file paths.
- Remove the "kie-sandbox" from the Staging Knative Workflow plugins artifacts.

### CODE REVIEW

**Review by paulovmr — APPROVED** [review]

File: packages/kn-plugin-workflow/pkg/common/helper.go [hunk 1]
```diff
@@ -21,12 +21,21 @@ import (
...
 	"time"
 
 	"github.com/briandowns/spinner"
 	"github.com/spf13/cobra"
 )
 
+func GetOsCommand(command string) string {
+	var osCommand = "./" + command
+	if runtime.GOOS == "windows" {
+		osCommand = ".\\" + command
```
Comment (barboras7) [inline packages/kn-plugin-workflow/pkg/common/helper.go]: I am not sure if I understand this correctly, however based on what I read here https://springhow.com/maven-wrapper-explained/, the mvnw command for windows should be in `mvnw.cmd` format. Based on our code, it's .\mvnw. Is it correct?

Comment (ljmotta) [reply packages/kn-plugin-workflow/pkg/common/helper.go]: Yes, it's correct. Strange, we've tested the `mvnw` for windows and we didn't have any problems. I'm going to make the change to use the `mvnw.cmd`.

File: packages/kn-plugin-workflow/pkg/common/helper.go [hunk 2]
```diff
@@ -100,3 +101,11 @@ func DefaultTemplatedHelp(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(cmd.ErrOrStderr(), "unable to display help text: %v", err)
 	}
 }
+
+func GetOsCommand(command string) string {
+	var osCommand = "./" + command
+	if runtime.GOOS == "windows" {
+		osCommand = ".\\" + command
```
Comment (barboras7) [inline packages/kn-plugin-workflow/pkg/common/helper.go:108]: This code returns `.\mvnw.cmd` for Windows, but based on what I found (for example in the article I posted here before) I would expect `mvnw.cmd`. Have you tested manually that the code works for Windows?

Comment (ljmotta) [reply packages/kn-plugin-workflow/pkg/common/helper.go:108]: Yes, I have. The `.\` says that the `mvnw.cmd` is in the current directory. This way works in the PowerShell (it doesn't work without it) and in the Command Prompt.

**Review by barboras7 — APPROVED** [review]
Looks good, thanks @ljmotta 👍

**Pull request merged**